src/modality-specific-files/physiological-and-other-continuous-recordings.md

```diff
@@ -18,40 +9,339 @@ specified using two files:
       -   [`7t_trt`](https://github.com/bids-standard/bids-examples/tree/master/7t_trt)
       -   [`ds210`](https://github.com/bids-standard/bids-examples/tree/master/ds210)
 
+## General specifications
+
+Continuous (that is, regularly sampled over time at a fixed frequency)
+physiological recordings such as cardiac and respiratory signals,
+asynchronous events corresponding to those signals, and other
+continuous measures (such as parameters of a film or audio stimuli)
+MAY be stored as tabular data in header-less
+[compressed tab-delineated (TSVGZ) files](../glossary.md#tsvgz-extensions).
+TSVGZ are formatted as TSV and MUST be compressed with
+[gzip](https://datatracker.ietf.org/doc/html/rfc1952).
+TSVGZ files MUST be accompanied by a JSON file with the same name as their
+corresponding tabular file but with a `.json` extension.
+Additional [metadata corresponding to TSVGZ files](../common-principles.md#tabular-files)
+MAY be included as prescribed by the common principles.
+For example, indicating units for recorded time series.
+
+??? warning "Columns of TSVGZ files MUST be defined in the corresponding JSON sidecar and the tabular content MUST NOT include a header line."
```

Comment:
```suggestion
???+ warning "Columns of TSVGZ files MUST be defined in the corresponding JSON sidecar and the tabular content MUST NOT include a header line."
```

probably better to use unfolded foldable admonitions so that users do not miss important information by "forgetting" to click on them.